virttest/tests/unattended_install.py

```diff
@@ -1157,6 +1157,38 @@ def string_in_serial_log(serial_log_file_path, string):
         return False
 
 
+def attempt_to_log_useful_files(test, vm):
+    """
+    Uses "vm.serial_console" to look in the system for useful log files
+    """
+    console = vm.serial_console
+    for path_glob in ["/*.log", "/tmp/*.log", "/var/tmp/*.log"]:
+        try:
+            paths = console.cmd("ls -1 %s" % path_glob)
+        except Exception as details:
+            continue
+        for path in paths.splitlines():
+            if not path:
+                continue
+            if path.startswith(os.path.sep):
+                rel_path = path[1:]
+            else:
+                rel_path = path
+            dst = os.path.join(test.outputdir, vm.name, rel_path)
+            dst_dir = os.path.dirname(dst)
+            if not os.path.exists(dst_dir):
+                os.makedirs(dst_dir)
+            with open(dst, 'wb') as fd_dst:
+                try:
+                    fd_dst.write(console.cmd("cat %s" % path))
+                    logging.info("Attached '%s' log file from guest", dst_dir)
```

Comment:
I think you meant `path` instead of `dst_dir`?

Reply:
Well I deliberately used `dst_dir` to let the user know where to find it (as not everyone is following the changes). If you really insist, I can change it, but IMO it's useful to see where the file is located in your results...

Reply:
@ldoktor then I guess you should've used `dst`, no?  `dst_dir` doesn't show the name of file, just the last dir.  I do find it more useful to list the file path *as it was* in the guest (that is, `path`) in addition to the location where it was saved.  That is, something like:

```python
base_dst_dir = os.path.join(test.outputdir, vm.name)
...
    logging.info('Attached "%s" log file from guest at "%s"', path, base_dst_dir)
```

Reply:
sure, sounds reasonable.